server/consumer.go

```diff
@@ -2955,7 +2961,7 @@ func (o *consumer) processWaiting(eos bool) (int, int, int, time.Time) {
 		wr := wq.reqs[rp]
 		// Check expiration.
 		if (eos && wr.noWait && wr.d > 0) || (!wr.expires.IsZero() && now.After(wr.expires)) {
-			hdr := []byte("NATS/1.0 408 Request Timeout\r\n\r\n")
+			hdr := []byte(fmt.Sprintf("NATS/1.0 408 Request Timeout\r\n%s: %v\r\n%s: %v\r\n\r\n", JSPullRequestPendingMsgs, wr.n, JSPullRequestPendingBytes, wr.b))
```

Comment:
Any reason to use `%v` for formatting `int` values instead of more descriptive compile-time safe `%d`?

Reply:
Good point. fixed.

Reply:
I have to say that I am guilty of using `%v` instead of `%d`...